Mathlib/RingTheory/Algebraic.lean

```diff
@@ -457,6 +457,46 @@ end Field
 
 end
 
+section
+
+variable {R S : Type*} [CommRing R] [Ring S] [Algebra R S]
+
+theorem IsAlgebraic.def_of_mem_nonZeroDivisors
```

Comment:
Why `def_`?

Reply:
Yeah, I guess it's not the best name. I was thinking that if there was an `IsAlgebraic.def`, it would say basically this but without the `coeff` part. I've renamed to `exists_nonzero_coeff_aeval` which hopefully communicates the content better.